packages/desktop-client/src/components/budget/MobileBudgetTable.js

```diff
@@ -248,203 +254,190 @@ function BudgetCategoryPreview({ name, pending, style }) {
   );
 }
 
-class BudgetCategory extends PureComponent {
-  constructor(props) {
-    super(props);
+const BudgetCategory = memo(function BudgetCategory({
+  category,
+  editing,
+  index,
+  // gestures,
+  editMode,
+  blank,
+  style,
+  month,
+  onEdit,
+  onBudgetAction,
+  show3Cols,
+  showBudgetedCol,
+}) {
+  // let opacity = new Animated.Value(editMode || blank ? 0 : 1);
+  let opacity = editMode || blank ? 0 : 1;
 
-    let { editMode, blank } = props;
-    this.opacity = editMode || blank ? 0 : 1;
-  }
+  // useEffect(() => {
+  //   if (editing && ACTScrollViewManager) {
+  //     ACTScrollViewManager.setFocused(findNodeHandle(container.current));
+  //   }
+  // }, [editing]);
 
-  render() {
-    let {
-      category,
-      editing,
-      index,
-      style,
-      month,
-      onBudgetAction,
-      show3Cols,
-      showBudgetedCol,
-    } = this.props;
-    let budgeted = rolloverBudget.catBudgeted(category.id);
-    let balance = rolloverBudget.catBalance(category.id);
-    let spent = rolloverBudget.catSumAmount(category.id);
-
-    let content = !category.hidden && (
-      <ListItem
+  // useEffect(() => {
+  //   Animated.timing(this.opacity, {
+  //     toValue: this.props.editMode ? 0 : 1,
+  //     duration: 200,
+  //     easing: Easing.inOut(Easing.ease),
+  //   }).start();
+  // }, [editMode]);
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏if it's not used - I'd recommend removing it.